javascript/organizing_your_javascript_code/organizing_code_with_objects.md

```diff
@@ -0,0 +1,374 @@
+### Introduction
+
+In our JavaScript fundamentals course, you should have learned the [basics of using objects](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/fundamentals-part-5) to store and retrieve data. Let's start with a little refresher.
+
+There are multiple ways to define objects, but in many cases **object literal** syntax is used as follows:
+
+```javascript
+const myObject = {
+  property: 'Value!',
+  otherProperty: 77,
+  "obnoxious property": function() {
+    // do stuff!
+  }
+};
+```
+
+There are also 2 ways to get information out of an object: dot notation and bracket notation.
+
+```javascript
+// dot notation
+myObject.property; // 'Value!'
+
+// bracket notation
+myObject["obnoxious property"]; // [Function]
+```
+
+Which method you use will depend on context. Dot notation is cleaner and is usually preferred, but there are plenty of circumstances when it is not possible to use it. For example, `myObject."obnoxious property"` won't work because that property is a string with a space in it. Likewise, you cannot use variables in dot notation:
+
+```javascript
+const variable = 'property';
+
+myObject.variable; // this gives us 'undefined' because it's looking for a property named 'variable' in our object
+
+myObject[variable]; // this is equivalent to myObject['property'] and returns 'Value!'
+```
+
+If you are feeling rusty on using objects, now might be a good time to go back and review the content in our [object basics lesson](https://www.theodinproject.com/lessons/foundations-object-basics) from our JavaScript Basics course.
```

Comment:
Got it. This was part of the original lesson, but it's been removed in my last commit addressing the previous items. 

If that changes your mind, let me know and it can be un-done.